quadcontrol/control.c

```diff
@@ -400,13 +446,23 @@ static int quad_configRead(void)
 		if ((line[0] == '@') && (strcmp(&line[1], "PID\n") == 0)) {
 			err = quad_pidParse(file, pidCnt++);
 			if (err < 0) {
+				printf("pids\n");
 				break;
 			}
 		}
 
 		/* @THROTTLE - define throttle min & max values */
 		if ((line[0] == '@') && (strcmp(&line[1], "THROTTLE\n") == 0)) {
 			err = quad_throttleParse(file);
+			if (err < 0) {
+				printf("throttle\n");
```

Comment:
```suggestion
				fprintf(stderr, "quad-control: cannot parse throttle parameters\n");
```
Suggestion applies to the rest of the conditions in `quad_pidParse` function.